src/App.test.tsx

```diff
@@ -60,8 +60,35 @@ describe("<App />", () => {
     expect(firstSuggestion).toHaveTextContent("Maczugi keczupowe");
     userEvent.click(firstSuggestion);
 
+    expect(await screen.findByText(/Treść problemu/i)).toBeInTheDocument();
+  });
+
+  it("navigates from detailed page to problems page after clicking on 'return to problems list' button", async () => {
+    testRenderer(<App />);
```

Comment:
philosophical question  - Shouldn't we write this one in polish 'return to problems list'?

Reply:
Good question, I would change it to polish, just to be precise. @franiakat what do you think? :)

Reply:
I'd remove the `'...'` and keep it in English, easier to maintain, as once we add more languages we won't need to change the it title. ;)